### PR TITLE
Add installation and configuration instructions for Rosetta

### LINKED ISSUE #770: Suggest new I18n shard
Currently, the internationalization section in the guide suggests using [vladfaust/i18n.cr](https://github.com/vladfaust/i18n.cr). That shard is lacking in functionality and it hasn't received an update in three years.

I'd suggest using [crystal-i18n/i18n](https://github.com/crystal-i18n/i18n) instead. It's pretty well-rounded, supports multi-file locales and even has decent documentation:

https://crystal-i18n.github.io/

### PR DESCRIPTION
**Pull request by wout**
This fixes #770 and https://github.com/luckyframework/lucky/issues/1284. I've run the setup through a brand new Lucky app and everything went well.

### CODE REVIEW

File: src/actions/guides/frontend/internationalization.cr [hunk 1]
```diff
@@ -9,133 +9,151 @@ class Guides::Frontend::Internationalization < GuideAction
...
-        not_in_system: "is not in our system"
-      user:
-        email: E-Mail
-        profile: Profile
-        next: "Next, you may want to"
-        auth_guides: "Check out the 'Authentication Guides'"
-        modify_page: "Modify this page"
-        after_signin: "Change where you go after sign in"
-        preferred_language: "Preferred Language"
+    ```cr
```
Comment (jwoertink) [inline src/actions/guides/frontend/internationalization.cr]:
```suggestion
    ```crystal
    # config/rosetta.cr
```

File: src/actions/guides/frontend/internationalization.cr [hunk 2]
```diff
@@ -9,133 +9,151 @@ class Guides::Frontend::Internationalization < GuideAction
...
+    Rosetta::Backend.load("config/rosetta")
     ```
 
-    ## Step 3 - Configure i18n within Lucky
-
-    The i18n needs to know the location of your language files.
-    > NOTE: This config file is only executed at load, so all language changes require a server restart.
+    Rosetta includes an integration macro for Lucky to include the
+    `Rosetta::Translatable` module everywhere translations are needed. Add the
+    following line to the initializer and you're good to go:
```
Comment (jwoertink) [inline src/actions/guides/frontend/internationalization.cr]: is this saying this command also need to go in `config/rosetta.cr`? If so, does it matter where?

File: src/actions/guides/frontend/internationalization.cr [hunk 3]
```diff
@@ -9,133 +9,151 @@ class Guides::Frontend::Internationalization < GuideAction
...
 
-    ```crystal
-    # config/i18n.cr
-    I18n.backend = I18n::Backends::YAML.new.tap do |backend|
-      backend.load_paths << Dir.current + "/config/locales"
-      backend.load
-    end
+    ```cr
```
Comment (jwoertink) [inline src/actions/guides/frontend/internationalization.cr]:
```suggestion
    ```crystal
```

File: src/actions/guides/frontend/internationalization.cr [hunk 4]
```diff
@@ -166,275 +184,272 @@ class Guides::Frontend::Internationalization < GuideAction
...
-        column lang : String
+        column language : String
         # ...
       end
       # ...
     end
     ```
 
-    ## Step 6 - Create a Translator module
+    ## Step 5 - Add a before pipe to you actions
```
Comment (jwoertink) [inline src/actions/guides/frontend/internationalization.cr]:
```suggestion
    ## Step 5 - Add a before pipe to your actions
```

**Review by jwoertink — CHANGES_REQUESTED** [review]
Sweet! This looks so nice. Thanks for doing this. I left a few comments and some small typo changes, but overall it looks pretty great!

File: src/actions/guides/frontend/internationalization.cr (hunk 2 above)
Comment (wout) [reply src/actions/guides/frontend/internationalization.cr]: Good point, it's a bit unclear so I'll fix that.

**Review by jwoertink — APPROVED** [review]
Love it! Great work on this :heart:

**Comment by wout** [comment]
Thanks :)

**Pull request merged**